core/src/main/resources/jenkins/model/experimentalflags/UserExperimentalFlagsProperty/config.jelly

```diff
@@ -0,0 +1,74 @@
+<!--
+The MIT License
+
+Copyright (c) 2022, CloudBees, Inc.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:f="/lib/form">
+    <j:set var="userProperty" value="${instance}"/>
+    <f:entry field="experimentalFlags">
+        <j:invokeStatic var="flagConfigs" className="jenkins.model.experimentalflags.UserExperimentalFlag"
+                        method="all"/>
+    
+        <j:if test="${empty(flagConfigs)}">
+            <div class="alert alert-info jenkins-form-item">
+                ${%NoFlagInfo}
+            </div>
+        </j:if>
+        <j:if test="${!empty(flagConfigs)}">
+            <f:rowSet name="flags" isFormItem="false">
+                <table class="jenkins-table sortable">
+                    <thead>
+                        <tr>
+                            <th>${%FlagKey}</th>
+                            <th>${%FlagDisplayName}</th>
+                            <th>${%FlagDescription}</th>
+                            <th>${%FlagConfig}</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <j:forEach var="flagConfig" items="${flagConfigs}" varStatus="loop">
+                            <j:set var="flagValue" value="${userProperty.getFlagValue(flagConfig.flagKey)}"/>
+
+                            <tr>
+                                <td id="flags_${flagConfig.flagKey}">
+                                    <a href="#flags_${flagConfig.flagKey}" title="${%Permalink}">#</a>
+                                    <st:nbsp/>
+                                    <code>${flagConfig.flagKey}</code>
```

Comment:
we could probably combine the columns with one of these options:

1. Display name (key)
2. Display name `<span class="jenkins-label jenkins-label--tertiary">key</span>`
<img width="447" alt="image" src="https://user-images.githubusercontent.com/21194782/197379951-f65275af-98e6-44de-b672-309ee783c2a4.png">
3. Show it only as a tooltip

Seems a bit unnecessary to display is as the first thing to users